nixos/modules/config/pulseaudio.nix

```diff
@@ -118,6 +118,13 @@ in {
           '';
         };
       };
+
+      displayName = mkOption {
```

Comment:
actually, I was completely wrong. I switched to directly using config.services.xserver.display, and also applied that fix to the unclutter service